experimental/vertx/image/Dockerfile-stack

```diff
@@ -0,0 +1,38 @@
+# Dockerfile for building the stack
+FROM adoptopenjdk/openjdk8-openj9
+
+USER root
+RUN  apt-get -qq update \
+  && apt-get -qq install -y curl maven \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq upgrade -y \
+  && apt-get -qq clean \
+  && rm -rf /tmp/* /var/lib/apt/lists/*
+RUN mkdir -p /.m2/repository
+
+COPY ./project /project
```

Comment:
We also need to copy the LICENSE and config here. Like this: 
<img width="205" alt="Screenshot 2019-08-29 at 16 17 23" src="https://user-images.githubusercontent.com/45264227/63953068-86ebd700-ca78-11e9-8563-8518abcbb8bb.png">

Reply:
right done